backend/app/models.py

```diff
@@ -282,6 +292,7 @@ class Document(models.Model):
     tokenized_text = models.JSONField(null=True, blank=True, default=None)
     word_count_counter = models.JSONField(null=True, blank=True, default=dict)
     part_of_speech_tags = models.JSONField(null=True, blank=True, default=list)
+    corpuses = models.ManyToManyField(Corpus)
```

Comment:
I really like this! This is a really neat way to keep track of documents in Corpora.